### ISSUE TITLE
extendWebpack error in app-vite using @next

### ISSUE THREAD

**Issue by maiolica** [quasarframework/app-extension-qpdfviewer#117]
Running

```bash
quasar ext add @quasar/qpdfviewer@next
````

from a freshly created Quasar project using `app-vite` cause this error during `quasar dev`:

```
TypeError: api.extendWebpack is not a function
```

It seems that an incorrect branch is being used, as the [proper check](https://github.com/quasarframework/app-extension-qpdfviewer/blob/v2.0.0-alpha.5/app-extension/src/index.js) for `app-vite` is not present in the fetched code.

Quasar info:

```
Operating System - Darwin(21.4.0) - darwin/arm64
NodeJs - 16.14.2

Global packages
  NPM - 8.5.0
  yarn - 1.22.17
  @quasar/cli - 1.3.2
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.6.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.0.0-beta.13 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.13.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.14
  pinia - Not installed
  vuex - 4.0.2 -- state management for Vue.js
  vite - 2.9.1 -- Native-ESM powered web dev build tool
  eslint - 8.12.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  @quasar/quasar-app-extension-qmarkdown - 2.0.0-beta.4 -- Display inline markdown in your Quasar App
  @quasar/quasar-app-extension-testing-e2e-cypress - 4.1.0 -- A Quasar App Extension for Cypress e2e
  @quasar/quasar-app-extension-qpdfviewer - 2.0.0-alpha.5 -- QPdfviewer is an app extension for viewing PDF files
```

**Comment by hawkeye64** [comment]
There is no support for app-vite, atm. Looking for help to PR this repo as I am currently slammed in my day job.

**Comment by gpproton** [comment]
Really looking forward to the vite support to0

**Comment by hawkeye64** [comment]
This is fixed with 2.0.0-alpha.6 release
